CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+0.10.2
+-----
+
+* Always upgrade `pip` for Python virtualenvs.
+* Install `wheel` in all Python virtualenvs.
```

Comment:
We were expecting it to be installed under Python 2, before. So I suspect this change is essentially a no-op for Python 2 & 3.

Reply:
Is it ok now?

Reply:
I don't think that's very clear: How about: Explicitly install `wheel` after upgrading pip.

Or just have no changelog entry for this.